class.jetpack-partner-subsidiary.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	// Exit if accessed directly.
+	exit;
+}
+
+use Automattic\Jetpack\Status;
+
+/**
+ * This class introduces routines to get a partner subsidiary id, that might be obtained from:
+ * - a `jetpack_partner_subsidiary_id` option in the WP database
+ * - a subsidiary id returned by a filter bound to the `jetpack_partner_subsidiary_id` filter hook
+ *
+ * @since 8.0.0
+ */
+class Jetpack_Partner_Subsidiary {
```

Comment:
Instead of adding this new class in Jetpack can we add it as a package? 
So that it could potentially be used by other plugins down the line?

Reply:
This makes sense. Currently, Jetpack Start only provisions Jetpack sites but we also side-load Akismet and VaultPress. We don't know if we are ever going to provision anything other than Jetpack but if we do it makes sense to put this in a place that can be used.